### PR TITLE
Update Polish (pl) localization of Appstores 160727

### PR DESCRIPTION
**Pull request by piotrdrag**
needs-review: @splewako

### CODE REVIEW

File: pl/whatsnew/whatsnew_android_48.lang [hunk 1]
```diff
@@ -18,6 +18,6 @@ Migracja listy „Do przeczytania” do zakładek
 
 # "Snazzy" is an informal English word that signifies "stylish", "attractive".
 ;Video controls get a new look! Snazzy
-Video controls get a new look! Snazzy
+Nowy wygląd elementów sterujących podczas odtwarzania filmów
```
Comment (piotrdrag) [inline pl/whatsnew/whatsnew_android_48.lang:21]: This is definitely UTF-8 on my computer. Clicking "View the whole file" on GitHub also shows correct characters.

Comment (splewako) [inline pl/whatsnew/whatsnew_android_48.lang:21]: New version looks good but let's double check after integrating to be sure.

Comment (piotrdrag) [inline pl/whatsnew/whatsnew_android_48.lang:21]: So that's a "go ahead", then?

Comment (splewako) [inline pl/whatsnew/whatsnew_android_48.lang:21]: Yes

**Pull request merged**